javagen/src/main/java/com/azure/autorest/mapper/ServiceClientMapper.java

```diff
@@ -125,7 +125,12 @@ public ServiceClient map(CodeModel codeModel) {
             String serviceClientPropertyDefaultValueExpression = serviceClientPropertyClientType.defaultValueExpression(p.getClientDefaultValue());
 
             if (settings.isLowLevelClient() && serviceClientPropertyName.equals("apiVersion")) {
```

Comment:
Do we need to test `settings.getServiceVersions() != null`?

Reply:
Yes, we do.